tox.ini

```diff
@@ -14,9 +14,9 @@ deps =
 [testenv]
 deps =
     py27-pylint14: astroid==1.4.6
-    py27-pylint14: pylint==1.5.6
-    py27-pylint20: git+https://github.com/PyCQA/astroid@461e016
-    py27-pylint20: git+https://github.com/PyCQA/pylint@0495355
+    py27-pylint14: pylint==1.6.4
+    py27-pylint20: git+https://github.com/PyCQA/astroid@master
```

Comment:
FYI I'm using a good newest sha without errors in our fork:
https://github.com/Vauxoo/pylint-odoo/blob/master/tox.ini#L25